src/index.ts

```diff
@@ -191,7 +197,7 @@ export default class MachinomyServerPlugin extends MiniAccountsPlugin {
     if (account.isBlocked) {
       throw new Error(`Cannot connect to blocked account ${account.accountName}`)
     }
-
+ 
```

Comment:
Looks like there's trailing whitespace here, is lint configured for this repo?

Reply:
Handled, thanks. It is -- didn't run the linter before pushing, will do so next time.